vrc-get-gui/locales/en.json5

```diff
@@ -278,10 +281,11 @@
     "vpm repositories:source:curated": "Curated",
     "vpm repositories:source:local": "User Local",
 
-    "packages:community repositories": "Community Repositories",
+    "packages:repositories": "Repositories",
     "vpm repositories:url": "URL",
     "vpm repositories:button:add repository": "Add Repository",
     "vpm repositories:remove repository": "Remove Repository",
+    "vpm repositories:remove curated or official repository": "You cannot remove curated or official repository",
```

Comment:
This should be "vpm repositories:tooltip:remove curated or official repository"?
It seems a tooltip.